data/card_catalogue/upgradeable-1.json

```diff
@@ -464,8 +267,8 @@
     "upgrade": "Reincarnate",
     "tier": "red",
     "details": {
-      "rules": "Heal 30  to a visible character",
-      "flavour": "\"I feel better than ever. Except for my body, which I can no longer feel\" - Hugo, Recently Decapitated",
+      "rules": "Restore a visible character to maximum health. Their maximum health permanently increases by 10 and they grow 3 feet taller.",
```

Comment:
I'm imagining a world where the players get this early on and use it every session, then eventually they're all absolute giants with unreal amounts of HP